compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/ContextMenuProvider.desktop.kt

```diff
@@ -251,7 +259,7 @@ class ContextMenuState {
             }
         }
 
-        object Closed : Status()
+        data object Closed : Status()
```

Comment:
This a change of API. We can no longer use:
```
status == ContextMenuState.Status.Closed
```

Reply:
Why not?

Reply:
My mistake, I read it as "data class" instead of "data object"